testsuite/layers/src/test/java/org/jboss/as/test/layers/LayersTestCase.java

```diff
@@ -114,6 +114,10 @@ public class LayersTestCase {
         "org.wildfly.security.jakarta.client.resteasy",
         "org.wildfly.security.jakarta.client.webservices",
         "org.jboss.resteasy.microprofile.config",
+        // TODO see gRPC proposal
+        "io.grpc",
```

Comment:
@hpehl FYI about this PR.

It removes the need for io.grpc from WF, but for WF 27 Beta I'll leave the module in place to not break your grpc feature pack. But for WF 27 Final I'd like to remove it and your FP would need to install it.

Reply:
@bstansberry thanks for letting me know!